### PR TITLE
Add link to the reset offset learning resource page

### PR DESCRIPTION
**Pull request by hemahg**
Add link to the reset offset learning resource page

### CODE REVIEW

File: ui/app/[locale]/(authorized)/kafka/[kafkaId]/consumer-groups/[groupId]/ResetOffsetModal.tsx [hunk 1]
```diff
@@ -69,11 +70,13 @@ export function ResetOffsetModal({
...
-        <StackItem>
-          <ExternalLink testId={"learn_to_shutdown_members"} href={""}>
-            {t("learn_to_shutdown_members")}
-          </ExternalLink>
-        </StackItem>
+        {isProductizedBuild && (
+          <StackItem>
+            <ExternalLink testId={"learn_to_shutdown_members"} href={"https://docs.redhat.com/en/documentation/red_hat_streams_for_apache_kafka/2.7/html-single/kafka_configuration_tuning/index#managing_offset_policy"}>
+              {t("learn_to_shutdown_members")}
+            </ExternalLink>
```
Comment (MikeEdgar) [inline ui/app/[locale]/(authorized)/kafka/[kafkaId]/consumer-groups/[groupId]/ResetOffsetModal.tsx:77]: The label and link should maybe go under `learning.labels` and `learning.links`. We should also have a link to something in the Apache Kafka documentation for upstream.

Comment (hemahg) [reply ui/app/[locale]/(authorized)/kafka/[kafkaId]/consumer-groups/[groupId]/ResetOffsetModal.tsx:77]: @MikeEdgar Is the phrase "Learn how to shut down members" correct to include under learning resources?

Comment (MikeEdgar) [reply ui/app/[locale]/(authorized)/kafka/[kafkaId]/consumer-groups/[groupId]/ResetOffsetModal.tsx:77]: @hemahg , yes that seems good.

Comment (riccardo-forina) [reply ui/app/[locale]/(authorized)/kafka/[kafkaId]/consumer-groups/[groupId]/ResetOffsetModal.tsx:77]: Isn't the wording a bit... aggressive? Not a native speaker here, but it feels that way

Comment (hemahg) [reply ui/app/[locale]/(authorized)/kafka/[kafkaId]/consumer-groups/[groupId]/ResetOffsetModal.tsx:77]: @shirimordechay, We've been discussing the phrasing for the learning resource, and I wanted to get some additional feedback. @MikeEdgar suggested "Learn how to stop consumers" as a more appropriate phrase, but I'd like to make sure everyone is on board with this change. Do you have any thoughts or suggestions for improving the wording or can help find relevant documentation links?

Comment (shirimordechay) [reply ui/app/[locale]/(authorized)/kafka/[kafkaId]/consumer-groups/[groupId]/ResetOffsetModal.tsx:77]: Rephrasing sounds good to me. I believe the doc link is the one @pmellor@redhat.com has provided, but I'm assuming we'll have a new link for GA. need to confirm that with Paul.

Comment (hemahg) [reply ui/app/[locale]/(authorized)/kafka/[kafkaId]/consumer-groups/[groupId]/ResetOffsetModal.tsx]: Update the phrase as per @MikeEdgar suggestion, there is no specific document to shutdown the consumer group members

**Comment by hemahg** [comment]
yes, we will close this PR, we can add the link when it's available

**Pull request closed without merging**